### PR TITLE
dbeaver/pro#1490 usage statistics

### PR DESCRIPTION
**Pull request by serge-rider**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageUsageStatistics.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.model.preferences.DBPPreferenceStore;
+import org.jkiss.dbeaver.model.runtime.features.DBRFeatureRegistry;
+import org.jkiss.dbeaver.runtime.DBWorkbench;
+import org.jkiss.dbeaver.ui.UIUtils;
+
+/**
+ * PrefPageQueryManager
+ */
+public class PrefPageUsageStatistics extends AbstractPrefPage implements IWorkbenchPreferencePage, IWorkbenchPropertyPage {
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageUsageStatistics.java]:
```suggestion
/**
 * PrefPageUsageStatistics 
 */
public class PrefPageUsageStatistics extends AbstractPrefPage implements IWorkbenchPreferencePage, IWorkbenchPropertyPage {
```

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java [hunk 2]
```diff
@@ -128,7 +127,8 @@ public boolean executeStatement()
         throws DBCException
     {
         try {
-            return SecurityManagerUtils.wrapDriverActions(connection.getDataSource().getContainer(), () -> execute(query));
+//            return SecurityManagerUtils.wrapDriverActions(connection.getDataSource().getContainer(), () -> execute(query));
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java:130]: Is this change intentional?

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java [hunk 3]
```diff
@@ -149,7 +149,8 @@ public void addToBatch() throws DBCException
     public int[] executeStatementBatch() throws DBCException
     {
         try {
-            return SecurityManagerUtils.wrapDriverActions(connection.getDataSource().getContainer(), this::executeBatch);
+//            return SecurityManagerUtils.wrapDriverActions(connection.getDataSource().getContainer(), this::executeBatch);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java:152]: And here

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java (hunk 2 above)
Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/exec/JDBCStatementImpl.java:130]: Yes. Eventually we decided to use another approach.

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageUsageStatistics.java (hunk 1 above)
Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageUsageStatistics.java]: Fixed

**Comment by MashaKorax** [comment]
verified
